### DISCUSSION TITLE
v4 beta: transition-[opacity,transform] stopped working

### DISCUSSION THREAD

**Discussion by dsine-de** [tailwindlabs/tailwindcss#15369] [Help] [answered]
Hi,

in v3 setting different transition properties like this works: `class="transition-[opacity,transform]"` - in v4 beta 6 however, this doesn't set the property correctly.

Should this be written differently?

**Comment by dsine-de** [comment]
Seems the transition properties are in fact being set correctly, but in v4 my example sets only one duration value.

**Comment by dsine-de** [comment]
Example in v4: https://play.tailwindcss.com/IYiNjIpplk
Example in v3: https://play.tailwindcss.com/K0vM6jRVTa

The v4 example has no transition duration for the `transform` property change.

**Reply by wongjn** [reply, marked as answer]
`translate-x-*` classes in v4 apply the `translate` property, not the `transform` property. Thus, you'd need to adjust the arbitrary `transition-*` class to match: https://play.tailwindcss.com/w5tPLw0fV6

**Reply by dsine-de** [reply]
Thanks!
